packages/rs-dpp/src/identity/state_transition/identity_credit_withdrawal_transition/apply_identity_credit_withdrawal_transition_factory.rs

```diff
@@ -34,37 +40,73 @@ where
         &self,
         state_transition: &IdentityCreditWithdrawalTransition,
     ) -> Result<()> {
-        let latest_withdrawal_index = self
+        let data_contract_id = Identifier::from_string(
+            &withdrawals_contract::system_ids().contract_id,
+            Encoding::Base58,
+        )?;
```

Comment:
Don't use base58 here, the system_ids should not be loaded as base58, they should be loaded as bytes so this conversion doesn't happen all the time.